library/std/src/process.rs

```diff
@@ -2318,10 +2318,8 @@ pub fn exit(code: i32) -> ! {
 /// Terminates the process in an abnormal fashion.
 ///
 /// The function will never return and will immediately terminate the current
-/// process in a platform specific "abnormal" manner.
-///
-/// Note that because this function never returns, and that it terminates the
-/// process, no destructors on the current stack or any other thread's stack
+/// process in a platform specific "abnormal" manner. Therefore,
```

Comment:
Just a small suggestion to make this sentence flow a little better.
```suggestion
/// process in a platform specific "abnormal" manner. Because of this,
```

Reply:
Isn't "therefore" syntax sugar for "because of this", which is more general in that "this" could be replaced with many other things, while "therefore" has no such flexibility?

Reply:
To me "therefore" puts the emphasis on the fact that no destructors are run, when that is more of a side effect of the main point. Having it as a separate sentence (like the current docs) also seems fine, because there are a couple more side effects that are listed in subsequent sentences.

"for this reason" would work as well. I'm only nitpicking because this entire PR is a bit of a nitpick and the current docs seem fine to me :)